cmake/configs/qurt_excelsior_default.cmake

```diff
@@ -8,5 +8,6 @@ include(configs/qurt_sdflight_default)
 
 # This definition allows to differentiate the specific board.
 add_definitions(
+	-D__PX4_QURT
```

Comment:
Are you sure this wasn't being set? It should be on for every QuRT target. https://github.com/PX4/Firmware/

Reply:
That's for excelsior board. It was missing before. It has been there for eagle board.

Reply:
That must be a bug in our cmake then, because it should be set for every QuRT board here: https://github.com/PX4/Firmware/blob/master/cmake/qurt/px4_impl_qurt.cmake#L175 (what I intended to link to above).

Reply:
I got it. I was wrong about it. Just updated again.

Reply:
Alright, we can do a pass later to remove the others if they're adding a define that's already there.